documentation/modules/rn-2.5.adoc

```diff
@@ -238,78 +245,81 @@ spec:
 
-.VM with multiple disks may boot from non-bootable disk after migration
-
-In previous releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
+.VM with multiple disks can boot from a non-bootable disk after migration
 
-This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated are able to boot on the target {ocp} cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
+In earlier releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
 
+This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated can boot on the target {ocp} cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
 
 .Transfer network not taken into account for cold migrations from vSphere
+
 In {project-short} releases 2.4.0-2.5.3, cold migrations from vSphere to the local cluster on which {project-short} was deployed did not take a specified transfer network into account. This issue is resolved in  {project-short} 2.5.4.  link:https://issues.redhat.com/browse/MTV-846[(MTV-846)]
 
-For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12424644[Resolved Issues] in Jira.
+.Fix migration of VMs with multi-boot guest operating system from vSphere 
+
+In {project-short} 2.5.6, the virt-v2v arguments include `–root first`, which mitigates an issue with multi-boot VMs where the pod fails. This is a fix for a regression that was introduced in {project-short} 2.4, in which the '--root' argument was dropped. link:https://issues.redhat.com/browse/MTV-968[(MTV-968)]
+
+.Errors logged in populator pods are improved
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by `must-gather` and by `forklift-controller` to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.npm IP package vulnerability
+
+A vulnerability found in the Node.js Package Manager (npm) IP Package can allow an attacker to obtain sensitive information and obtain access to normally inaccessible resources. link:https://issues.redhat.com/browse/MTV-941[MTV-941]
+
+This issue has been resolved in {project-short} 2.5.6.
+
+For more information, see link:https://access.redhat.com/security/cve/cve-2023-42282[CVE-2023-42282]
+
+For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12431371[Resolved Issues] in Jira.
```

Comment:
I've noticed that a fix for CVE-2023-39326 [MTV-850](https://issues.redhat.com/browse/MTV-850) is also included in 2.5.6, can you please add it as well?

Reply:
Added along with the other changes

Reply:
sure, no need for the non-public Jira issue, thanks